package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "dotenv": "^16.0.2",
     "fastboot-app-server": "^3.3.2",
-    "node-fetch": "^3.2.10"
+    "node-fetch": "2"
```

Comment:
Why is the version low here

Reply:
We are using Fastboot and it requires CommonJS imports to be used therefore the stable and still running version of node-fetch with ESM as well as CommonJS support is version 2
https://github.com/node-fetch/node-fetch#loading-and-configuring-the-module
![image](https://user-images.githubusercontent.com/78433013/210217825-f159423c-5ee8-4f45-b2ef-fff9a7b35a0b.png)